sdk/formrecognizer/azure-ai-formrecognizer/src/test/resources/session-records/clientBuilderWithHttpEndpoint.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Is this test for an invalid Http pipeline?

Reply:
move this test into BuilderUnitTest, Azure core requires https protocal: "Key credentials require HTTPS to prevent leaking the key."